README.md

```diff
@@ -66,20 +73,20 @@ Quick Start on localhost
 
 * Index the web app's UI static files (css, js, jpeg etc)
 ```
-    python ./oauth2server/manage.py collectstatic --noinput
+    pipenv run ./oauth2server/manage.py collectstatic --noinput
 ```
 
 * Start the server on your dev default port with either;
 
-- - the django development web server
-```
-    oauth2server/manage.py runserver 0.0.0.0:8040
-```
+    * the django development web server
+    ```
+        pipenv run oauth2server/manage.py runserver 0.0.0.0:8040
+    ```
 
-- -  or the gunicorn web server
-```
-    gunicorn --bind 0.0.0.0:8040 --workers 4 proj.wsgi --pythonpath 'oauth2server'
-```
+    * or the gunicorn web server
+    ```
+        gunicorn --bind 0.0.0.0:8040 --workers 4 proj.wsgi --pythonpath 'oauth2server'
```

Comment:
Does `pipenv run` need to be appended here as well?

Reply:
yep :+1: